src/keyDerivation.c

```diff
@@ -79,32 +79,32 @@ void deriveRawPublicKey(
 
 void extractRawPublicKey(
         const cx_ecfp_public_key_t* publicKey,
-        uint8_t* rawPublicKey, size_t rawPublicKeySize
+        uint8_t* outBuffer, size_t outSize
 )
 {
 	// copy public key little endian to big endian
-	ASSERT(rawPublicKeySize == 32);
+	ASSERT(outSize == 32);
 	uint8_t i;
 	for (i = 0; i < 32; i++) {
-		rawPublicKey[i] = publicKey->W[64 - i];
+		outBuffer[i] = publicKey->W[64 - i];
```

Comment:
STATIC_ASSERT(SIZEOF(publicKey->W) == 65) ? (Note: This is either 65 or buffer overflow :-))